### ISSUE TITLE
Win11 testing: wspy quits after "Spy" -> "Select Window ..."

### ISSUE THREAD

**Issue by winspool** [open-watcom/open-watcom-v2#1170]
After starting `binnt64\wspy.exe` and  selecting the nenu
 `Spy` -> `Select Window ...`, the `wspy` program quits.

When using `binnt\wspy.exe` and  selecting the menu
`Spy` -> `Select Window ...`, the main `wspy` window is hidden
and a helper Window `Select Window` is opened.

.. 
Regards ... Detlef

**Comment by joncampbell123** [comment]
@jmalak I'm trying to get the ow source tree to compile wspy but it won't build wspy. The host is Linux x86_64. I understand that even from Linux the build still also makes Windows NT and NT 64-bit binaries.

**Comment by jmalak** [comment]
You cannot cross compile 64-bit executable.
You can compile only for host system because it use host native compilers instead of OW compilers.

**Comment by joncampbell123** [comment]
Right, but it also does not make the winnt i386 32-bit build either.

That's unfortunate, I was hoping to debug this issue.

**Comment by jmalak** [comment]
32-bit cross-compile by bootstrap compiler should work.
Could you give me build log part for issue.

**Comment by joncampbell123** [comment]
I'll rebuild from scratch again. Once it's built, "builder rel" will not build the spy program no matter what. No error is provided, the build log shows nothing, but that's after bootstrapping.

**Comment by jmalak** [comment]
OK. I will check on my Linux box why it happen.
May be related to some last big changes in main build make files.
I consolidated two make files bhost.mif and local.mif to local.mif.
What happen if go into bld/sdk/spy/nt386 and run **`wmake clean`** and next **`wmake`**?

**Comment by jmalak** [comment]
On my Ubuntu Linux box it works and build nt386 and wini86 version of Spy.

**Comment by joncampbell123** [comment]
Does not build here. GCC 9.3, custom Linux from scratch system. I also cannot build idedemo because it does not build 'bide2mak' or something of that name.

**Comment by jmalak** [comment]
Check your OW build environment script to not have setup following environment variables which exclude some project build.
# export OWNOWGML=1                                                                                                                                                                                                                                # export OWGUINOBUILD=1    
or if you use 
# export OWNOBUILD=

**Comment by winspool** [comment]
I rechecked with the current ci build,
but `binnt64\wspy.exe` still quits after selecting the menu
`Spy` -> `Select Window ...`
